src/components/AppDialog/AppDialog.types.ts

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export type AppDialogProps = {
+    openDialog: boolean,
```

Comment:
Just discussion: I think we should agree to name booleans like `isOpenDialog`
```suggestion
    isOpenDialog: boolean,
```